### DISCUSSION TITLE
Tractor engine speed PTO vs speed at heavy load

### DISCUSSION THREAD

**Discussion by vaclavikzahradka** [Mogli12/KeyboardSteer#774] [Ideas]
First I have to thank Mogli12 for the excellent mods, without which I would probably give up FS22. Your mods are professional, at a high level and thank you very much for that. Above all, the gearbox mod, which we have thanks to you since FS17. With FS22 you can see that you are improving the mod and that is good. I am very happy that you have combined (synchronized) the engine speed when using the PTO with the tractor speed. I would also appreciate that when using a machine connected to a tractor that requires PTO shaft work (such as a lawn mower), if the tractor on a hill does not tighten the engine power, the engine speed and, consequently, the tractor speed will start to run in parallel. Then you need to engage a lower gear so that the tractor can continue working. This is how it works in this mode and in FS19. Thank you for your fashion. I apologize for the inaccuracies in the text, I use the Google translator. David. 

Zuerst muss ich Mogli12 für die hervorragenden Mods danken, ohne die ich FS22 wahrscheinlich aufgeben würde. Eure Mods sind professionell, auf hohem Niveau und vielen Dank dafür. Allen voran der Getriebe-Mod, den wir euch seit FS17 zu verdanken haben. Mit FS22 kannst du sehen, dass du die Mod verbesserst und das ist gut so. Ich freue mich sehr, dass Sie die Motordrehzahl bei Nutzung der Zapfwelle mit der Schlepperdrehzahl kombiniert (synchronisiert) haben. Ich würde es auch begrüßen, wenn bei der Verwendung einer Maschine, die an einen Traktor angeschlossen ist, der Zapfwellenarbeit erfordert (z beginnen parallel zu laufen. Dann müssen Sie einen niedrigeren Gang einlegen, damit der Traktor weiterarbeiten kann. So funktioniert es in diesem Modus und in FS19. Danke für deine Mode. Ich entschuldige mich für die Ungenauigkeiten im Text, ich benutze den Google-Übersetzer. David.

**Comment by Mogli12** [comment]
Did you turn on the setting "Modify Motor RPM"?

Hast Du die Motormodifikationen in den Einstellungen eingeschaltet?

**Comment by Mogli12** [comment]
Please try out build 42.

**Comment by vaclavikzahradka** [comment]
Perfect, you are just an excellent programmer, thank you very much for improving the mode. Build 41 solved this problem, now the engine speed (PTO) and the tractor speed are synchronized. During testing, I noticed that using the machine with a PTO does not use maximum engine power when the machine (mower) is switched on and the tractor is running. It is necessary to depress the accelerator pedal, then the engine delivers the highest power and everything works fine. This is different from the FS19, where the engine was able to deliver the highest possible power on its own when using the PTO, without having to depress the accelerator pedal. But maybe it's your intention, maybe it's set up on purpose. 
Build 42 does not work properly, the tractor with a manual transmission does not run faster than approximately 1200 RPM when driving (on the road). Also when using a PTO (field) when the tractor is in motion. Everything is fine with a tractor with a CVT transmission. 

Perfekt, du bist einfach ein ausgezeichneter Programmierer, vielen Dank für die Verbesserung des Modus. Build 41 hat dieses Problem gelöst, jetzt sind die Motordrehzahl (PTO) und die Traktordrehzahl synchronisiert. Beim Testen ist mir aufgefallen, dass die Verwendung der Maschine mit Zapfwelle bei eingeschalteter Maschine (Mähwerk) und laufendem Traktor nicht die maximale Motorleistung verbraucht. Das Gaspedal muss durchgetreten werden, dann liefert der Motor die höchste Leistung und alles funktioniert einwandfrei. Anders als beim FS19, wo der Motor bei Nutzung der Zapfwelle selbstständig die höchstmögliche Leistung erbringen konnte, ohne das Gaspedal durchtreten zu müssen. Aber vielleicht ist es Ihre Absicht, vielleicht ist es mit Absicht eingerichtet. 
Build 42 funktioniert nicht richtig, der Traktor mit Schaltgetriebe läuft beim Fahren (auf der Straße) nicht schneller als ca. 1200 U/min. Auch bei Verwendung einer Zapfwelle (Feld) während der Traktor in Bewegung ist. Bei einem Traktor mit CVT-Getriebe ist alles in Ordnung.

**Comment by vaclavikzahradka** [comment]
To Build 43 I will write - Stefan you are God! Perfect work, now everything works exactly according to the best ideas. Thank you, thank you, thank you. Giants developers should learn from you how to do things right. You make the game real. 

Zu Build 43 schreibe ich - Stefan, du bist Gott! Perfekte Arbeit, jetzt funktioniert alles genau nach den besten Ideen. Danke danke danke. Giants-Entwickler sollten von Ihnen lernen, wie man die Dinge richtig macht. Du machst das Spiel real.
